iidm/iidm-api/src/main/java/com/powsybl/iidm/network/ValidationUtil.java

```diff
@@ -66,6 +77,16 @@ public static void checkActivePowerLimits(Validable validable, double minP, doub
         }
     }
 
+    /**
+     * @deprecated
```

Comment:
This remark is valid for all other deprecated methods in here from what I can see: why did you deprecate this method? It was implemented to do some check without checking everything when only one regulation parameter was changed (here targetDeadband). Not sure I understand why it shouldn't be used? I am asking since this change could cause issues in the future for other implementations using the "old" methods to keep track on checks.

Reply:
I decided to organize the validation methods in two sets: 

- The complete check methods that verify all the attributes associated with the regulating control.
- The individual check methods that only verify one attribute. 

Complete methods are used in the adder and when  the enabled attribute is changed after doing the adder. If the regulating control is enabled, all the attributes must be valid, complete verification.

Individual methods are used when the corresponding attribute is changed after the adder. The verification of one attribute depends always on the enabled attribute. It the control is enabled only valid attributes are allowed. 

If the control is not enabled the verification is not done. That allows to set the attributes of the controls after the adder. (Controls must be filled at the end of the conversion process so all remote regulating terminals have previously been defined). When only the EQ file is imported the control could be created disabled with not valid attributes.